doc/reference/trd-syscalls.md

```diff
@@ -200,19 +200,20 @@ are additional error codes to include errors related to userspace.
 | 13    | NOACK       | The packet transmission was sent but not acknowledged.                                  |
 | 1024  | BADRVAL     | The variant of the return value did not match what the system call should return.       |
 
-Any value not specified in the above table is reserved.
+Any value not specified in the above table is reserved and it
+MUST NOT be used.
```

Comment:
I personally found the wording "Any value not specified in the above table is reserved." a bit ambiguous. The way I read it, it was "reserved for future use" -- a future kernel version could use a different values, and processes must be ready to handle the unexpected value.

The way I read your new wording, you are saying that a Tock 2.0 kernel will *never* return a different value, which allows for a different `libtock-rs` implementation.

I do not expect that you intended this to be a semantics change, because if it is a semantics change it seems orthogonal to this PR. Should this be in a different PR?

Reply:
Oh, this was actually part of #2351 .

Reply:
Phil, is there a merge or rebase that should be done to remove changes than have already been merged?